README.md

```diff
@@ -1,7 +1,17 @@
-# starter-kit
+# WICG Starter kit
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/WICG/starter-kit.svg)](https://greenkeeper.io/)
-A simple starter kit for [WICG](https://wicg.io) specs.
+Welcome!  This is a simple starter kit for writing [WICG](https://wicg.io) specifications, and adding new awesome features to the web platform.  In this repo you'll find templates and guides for using them.
+
+## Process
+
+Start with an explainer. [Learn more](#TODO-slightlylate) and [get the template](https://github.com/WICG/starter-kit/blob/master/templates/explainer.md).  The explainer is critical to getting early engagement and understanding about what you'll be building.
+
+When you move into writing the spec itself, use a document framework such as [ReSpec](https://github.com/w3c/respec) or [Bikeshed](https://github.com/tabatkins/bikeshed).  This will ensure that you apply the appropriate metadata and provide details to help people locate the spec and understand the structure.
```

Comment:
@marcoscaceres you sir, are awesome -- I told you that already, but worth repeating. Ping me when you have something, happy to review. 

The reason I brought this up is because we were also discussing the need for such a resource in this weeks webperf WG call: a common reference to point new spec authors, etc.